types/google-cloud__kms/index.d.ts

```diff
@@ -0,0 +1,275 @@
+// Type definitions for @google-cloud/kms 0.2
+// Project: https://github.com/googleapis/nodejs-kms
+// Definitions by: Ben Talbot <https://github.com/ben-tbotlabs>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+/// <reference types="node" />
```

Comment:
I might be missing something, but was there supposed to be an export somewhere?

Reply:
I confess I was confused about how to mark the exports since the source at (https://github.com/googleapis/nodejs-kms/blob/master/src/index.js) is

// Import the clients for each version supported by this package.
const gapic = Object.freeze({
  v1: require('./v1'),
});

/**
 * @type {object}
 * @property {constructor} KeyManagementServiceClient
 *   Reference to {@link v1.KeyManagementServiceClient}
 */
module.exports = gapic.v1;

/**
 * @type {object}
 * @property {constructor} KeyManagementServiceClient
 *   Reference to {@link v1.KeyManagementServiceClient}
 */
module.exports.v1 = gapic.v1;

// Alias `module.exports` as `module.exports.default`, for future-proofing.
module.exports.default = Object.assign({}, module.exports);

and when adding export to class KeyManagementServiceClient I got there error 

'export' keyword is redundant here because all declarations in this module are exported automatically. If you have a good reason to export some declarations and not others, add 'export {}' to the module to shut off automatic exporting. See: https://github.com/Microsoft/dtslint/blob/master/docs/strict-export-declare-modifiers.md

Help here would definitely be appreciated.

Reply:
So it turns out that everything inside a `declare module` is implicitly exported by default, which is why you get the lint error when you mark it explicitly.

I think you should drop the surrounding `declare module '@google-cloud/kms { ... }` &mdash; at least, as long as the tests still pass without it. I think your path mapping in the tsconfig will take care of converting "@google-cloud/kms" to "google-cloud__kms".